### ISSUE TITLE
Smoother experience for TypeScript

### ISSUE THREAD

**Issue by danielearwicker** [fuse-box/fuse-box#140]
Just a couple of things that would make it "just work" for more users:

1. Default behaviour for finding tsconfig.json is to look in home folder. Maybe search up the folder hierarchy?

2. Automatically include TypeScriptHelpers plugin if needed.

**Comment by nchanged** [comment]
Thanks! We actually moved this issue on top of everything else. Seems like people are complaining about it.

**Comment by Grafikart** [comment]
Additional problem, I write the following code 

```
class Demo {

  greet (n: number) {
    console.log(3 * n)
  }

}

let demo = new Demo()
demo.greet("Hello marc") // NaN
```

I would expect the build to detect the problem and throw an error (or at least show something in the console). tsc throw something like `error TS2345: Argument of type '"Hello marc"' is not assignable to parameter of type 'number'`

**Comment by nchanged** [comment]
Automatic `tsconfig.json` lookup has been implemented in `1.3.110`. Let's create a different issue regarding typescript helpers and a type checker daemon
